draft-ietf-rats-corim.md

```diff
@@ -1590,20 +1596,22 @@ Profile (label 6):
 
 : The profile that defines this tuple. If no profile is used, this attribute is omitted.
 
+The following CDDL describes the ECT structure in more detail.
+
 ~~~ cddl
 ECT = {
-  ? e: environment-map
-  ? c: claims-map / [ + local-claim ]
-  ? a: [ + $crypto-key-type-choice ]
-  ? ns: text
-  ? cm: cm-type
-  ? p: $profile-type-choice
+  ? environment: environment-map
+  ? properties: [ + properties-map ]
```

Comment:
Given that `ECT` is an acronym for `Environment Claims Tuple`: I would rename line 1604 on the right to 

? iclaims: [+internal-claims-map]

&

Line 1610 to be:
internal-claims-map = {
? eid: element-id
claims: measurement-values-map
}

Reply:
I keyed off of the existing prose that describes it as properties. I don't think we need to bow to the consistency gods by changing ECT to EPT since the main focus is still on claims. In other words, the measurement-values-map contents are still the "claims". 

I find it easier to write the prose if the structures are all the same but with slight modification. 

Given we now describe environments as consisting of zero or more elements. It may be appropriate to name `properties-map` to `element-map` then the contents of an element-map is the element-id and the claims `claims-map`.

Reply:
Works for me and I admit a very sensible suggestion.